### DISCUSSION TITLE
Calculating area-weighted fire intensity from FATES output

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#979] [Q&A] [answered]
Hi all,
I recently outputted FATES_BURNFRAC, FATES_FIRE_INTENSITY, and FATES_FIRE_INTENSITY_BURNFRAC for a single-site simulation at a lower montane Sierra Nevada site with conifer, oak, and shrub pfts. The variable description for FATES_FIRE_INTENSITY_BURNFRAC say's that if you divide by FATES_BURNFRAC you can get the area-weighted mean fire intensity. When I do this I get fire intensities as high as 828,773,760,000 (8e11) W m-1 (8e8 kW m-1) which I'm 99% sure are unrealistically high.

Perhaps the issue is that FATES_BURNFRAC has units of s-1 and the [calculation for FATES_FIRE_INTENSITY_BURNFRAC](https://github.com/NGEET/fates/blob/main/main/FatesHistoryInterfaceMod.F90#L3527) is FIRE_INTENSITY x the fractional area of the fire, which I suppose has an implicit time dimension of something other than 1 second. How long can fires burn in FATES? 

Assuming a fire burns for a day in FATES (probably should be longer?), I'm thinking that area-weighted fire intensity would actually be calculated as FATES_FIRE_INTENSITY_BURNFRAC / (FATES_BURNFRAC * 86400 s day-1).

This would give more reasonable results. Max fire intensity of ~9,000 kW m-1

Does this make sense to others? Do we assume fires burn for one day or longer?
Thanks for any input!
Adam

**Comment by ckoven** [comment, marked as answer]
Hi @adamhb, yes you are correct.  `FATES_BURNFRAC` used to have units of day<sup>-1</sup> , but now (as of https://github.com/NGEET/fates/pull/802) has units of sec<sup>-1</sup>. But as you point out, the day<sup>-1</sup>  to sec<sup>-1</sup> conversion wasn't also applied to `FATES_FIRE_INTENSITY_BURNFRAC`.  So the variable long name is now incorrect, one needs to divide by 86400.

**Reply by adamhb** [reply]
Thanks for confirming @ckoven !

**Comment by jkshuman** [comment]
good catch @adamhb thanks for opening this.
